### PR TITLE
ref: don't allow unsighed values on session

### PR DESCRIPTION
**Pull request by bruno-garcia**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ## 5.0.4
 
 - fix: End file at the right place with #ifdef #521
+- ref: Session values are unsigned #527
```
Comment (philipphofmann) [inline CHANGELOG.md:8]: `m`: Shouldn't this be in `## unreleased`

File: Sources/Sentry/include/SentrySession.h [hunk 2]
```diff
@@ -23,8 +23,8 @@ typedef NS_ENUM(NSUInteger, SentrySessionStatus) {
 @property (nonatomic, readonly, strong) NSUUID *sessionId;
 @property (nonatomic, readonly, strong) NSDate *started;
 @property (nonatomic, readonly) enum SentrySessionStatus status;
-@property (nonatomic, readonly) NSInteger errors;
-@property (nonatomic, readonly) NSInteger sequence;
+@property (nonatomic, readonly) NSUInteger errors;
```
Comment (philipphofmann) [inline Sources/Sentry/include/SentrySession.h:26]: I like this change 👏

File: CHANGELOG.md (hunk 1 above)
Comment (bruno-garcia) [reply CHANGELOG.md:8]: good catch! thanks

**Comment by bruno-garcia** [comment]
Thanks @philipphofmann for picking up on that changelog mistake

**Pull request merged**